src/main/kotlin/de/cyface/uploader/DefaultUploader.kt

```diff
@@ -355,14 +368,14 @@ class DefaultUploader(private val apiEndpoint: String) : Uploader {
 
             // Location meta data
             if (metaData.startLocation != null) {
-                attributes["startLocLat"] = metaData.startLocation.latitude.toString()
-                attributes["startLocLon"] = metaData.startLocation.longitude.toString()
-                attributes["startLocTS"] = metaData.startLocation.timestamp.toString()
+                attributes["startLocLat"] = metaData.startLocation!!.latitude.toString()
```

Comment:
Does this not compile without the `!!`? Usually Kotlin should know that `!!` is not necessary after a check for `null`. Why was this not there previously?